doc/api/errors.md

```diff
@@ -250,7 +250,7 @@ not capture any frames.
 
 #### error.message
 
-Returns the string description of error as set by calling `new Error(message)`.
+Is the string description of error as set by calling `new Error(message)`.
 The `message` passed to the constructor will also appear in the first line of
 the stack trace of the `Error`, however changing this property after the
 `Error` object is created *may not* change the first line of the stack trace.
```

Comment:
We should be clear here. I know this was already in the doc, but *may not* is not that clear. We should say whether it will or will not or indicate the conditions under which it may or may not.